src/components/Tool/Appeal/InitialPage/AppealsInitialPage.tsx

```diff
@@ -66,7 +69,19 @@ const AppealsInitialPage: React.FC = () => {
                   })}
                 />
                 <CardContent>
-                  <AddAppealForm accountListId={accountListId || ''} />
+                  <AddAppealForm
+                    accountListId={accountListId || ''}
+                    appealStatuses={[
+                      {
+                        name: '-- All Active --',
+                        value: 'ACTIVE',
+                      },
+                    ]}
+                    appealExcludes={contactExclusions.filter(
+                      (exclusion) =>
+                        exclusion.value === ExclusionEnum.DoNotAskAppeals,
+                    )}
+                  />
```

Comment:
These changes need to be added to the form itself as this is the default.

Can you move these changes inside the `<AddAppealForm/>`?

Reply:
It accepts props as they will be used in the announcements, which can pass down appeal statements and appeal excludes.

Can we make the appealStatuses and appealExcludes have a default value of what you have above?

Reply:
It uses it in the old one here: `src/tools/appeals/appeals.service.ts` Line 62 "oneClickAppeal"